### ISSUE TITLE
Add a Helper Package for Manifest Testing

### ISSUE THREAD

**Issue by crhntr** [pivotal-cf/kiln#356]
TL;DR: may I mv `github.com/pivotal-cf/planitest` to `github.com/pivotal-cf/kiln/pkg/planitest`

## Context

Kiln will soon have a test command to execute manifest tests.

`github.com/pivotal-cf/planitest` is currently proprietary Go package but just wraps other CLIs and provides some small convenience functions. It wraps [om](https://github.com/pivotal-cf/om) (open source) or ops-manifest (proprietary, internal only). It is used for TAS tile development and is used in hello-tile.

## Problem

As implemented, once the `kiln test` feature is merged kiln will fail to compile for non non-vmware employeees because `go mod download` will fail.

Also `kiln test` acceptance tests require the closed source planitest package to run properly.

We have already "exposed" planitest in a feature branch vendor directory.

## Solution

- Move the planitest package to kiln with the import path `github.com/pivotal-cf/kiln/pkg/planitest`
- Document that the om renderer was/is broken.
- Archive the existing `github.com/pivotal-cf/planitest` repository
- Make PRs to tiles (tas, ist, tasw, and hello-tile) using planitest to use the Kiln version

**Comment by crhntr** [comment]
I've done the work locally but am waiting for approval to push the code.

**Comment by anEXPer** [comment]
This is, implicitly, making ownership of the code by the maintainers of kiln explicit. If that is intended and acceptable, I'm down to bring it in.

If, upon archiving the other repo, we learn of other users of the code, that would be worth learning.

This seems fine to me.
